app/controllers/users.js

```diff
@@ -0,0 +1,31 @@
+const { logger } = require('express-wolox-logger');
+const { errors, statusCodes } = require('../helpers');
+const { userMapper } = require('../mappers');
+const { UserService } = require('../services');
+
+const signUp = async (req, res, next) => {
+  try {
+    const userDTO = userMapper.signUpDTO(req.body);
+    const dataUser = await UserService.getUser({ email: userDTO.email });
+    if (dataUser !== null) {
```

Comment:
```suggestion
    if (dataUser) {
```
in JS there are trusty (eg. object) and falsy values (eg. null),  so I don't think you need to create a  value !== null